src/types.ts

```diff
@@ -154,3 +160,8 @@ export type ValidationType =
   | 'Array';
 
 export type Value = AstType | Scalar;
+
+export type ParserArgs = {
+  file?: string;
+  slots?: boolean;
```

Comment:
I'm usually a little wary of adding boolean config options like this. WDYT about using an enum instead? Like `slotMode: 'enabled'` or something?

Reply:
I prefer the boolean for simplicity and because it's consistent with how tokenizer options work.